src/Fixer/Basic/BracesFixer.php

```diff
@@ -869,37 +881,6 @@ private function getFinalControlContinuationTokensForOpeningToken(int $openingTo
         return [];
     }
 
-    private function fixDeclareStatement(Tokens $tokens, int $index): void
-    {
-        $tokens->removeTrailingWhitespace($index);
-
-        $startParenthesisIndex = $tokens->getNextTokenOfKind($index, ['(']);
-        $tokens->removeTrailingWhitespace($startParenthesisIndex);
-
-        $endParenthesisIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $startParenthesisIndex);
-        $tokens->removeLeadingWhitespace($endParenthesisIndex);
-
-        $startBraceIndex = $tokens->getNextTokenOfKind($endParenthesisIndex, [';', '{']);
-        $startBraceToken = $tokens[$startBraceIndex];
-
-        if ($startBraceToken->equals('{')) {
-            $this->fixSingleLineWhitespaceForDeclare($tokens, $startBraceIndex);
-        }
-    }
-
-    private function fixSingleLineWhitespaceForDeclare(Tokens $tokens, int $startBraceIndex): void
-    {
-        // fix single-line whitespace before {
```

Comment:
what about this case?

or you want to have 2 fixers around `declare` keyword, one for whitespace around `(` and one for `{` ?

Reply:
This handling of `{` is similar to how we treat it elsewhere so yes, I would delegate that to another fixer.

By the way, the code removed here was not moved in the new fixer but `braces` tests still pass, which means this code was likely not needed.